elastic/kibana
# [Reporting] use docLinks service in reporting

## Summary

Closes https://github.com/elastic/kibana/issues/127026

## Testing
1. Create build of Kibana from this branch with `yarn build --skip-os-packages`
2. Start a minimal Docker Ubuntu environment:
    ```
    docker run \
     -it \
     --mount type=bind,source="$(pwd)"/build,target=/build,readonly \ # place the build in the `build` directory
     --publish 127.0.0.1:5601:5601/tcp \
     ubuntu:latest
    ```
    kibana.yml:
    ```
    server.host: 0.0.0.0
    elasticsearch.hosts: [http://gateway.docker.internal:9200]
    elasticsearch.username: kibana_system
    elasticsearch.password: changeme 
    xpack.reporting.roles.enabled: false
    xpack.reporting.encryptionKey: oasgjasd0as7lurkjgserlclkurwayhtcluaovasjuslekr
    ```
3. Test the diagnostic tool in Stack Management > Reporting. It should show an error about missing OS dependencies, with a URL from the Docs Links service:
    <img width="1774" alt="image" src="https://user-images.githubusercontent.com/908371/181640339-86f50688-4405-46f8-ac28-30e26533a105.png">


### Checklist

Delete any items that are not applicable to this PR.
